docs/sections/user_guide/yaml/tags.rst

```diff
@@ -0,0 +1,76 @@
+.. _defining_YAML_tags:
+
+Defining YAML ``tags``
+==========================
+
+Tags are used to denote the type of a YAML node when converting to a Python object. Standard YAML tags are defined at http://yaml.org/type/index.html.
+
+Tags may be implicit:
+
+.. code-block:: yaml
+
+   boolean: true
+   integer: 3
+   float: 3.14
+
+Or explicit:
+
+.. code-block:: yaml
+
+   boolean: !!bool "true"
+   integer: !!int "3"
+   float: !!float "3.14"
+
+Additionally, UW extends the following tags and constructors to support other formats:
```

Comment:
```suggestion
Additionally, UW defines the following tags to support use cases not covered by standard tags:
```
I appreciate these docs and how they're making me think about what's really going on. For example, the `!!int`` standard tag doesn't work here:
```
% echo 'n: !!int "{{ 2 + 2 }}"' | uw config realize --input-format yaml --output-format yaml
...
ValueError: invalid literal for int() with base 10: '{{ 2 + 2 }}'
```
because `pyyaml` tries to convert the Jinja2 expression immediately, but our custom tag works:
```
% echo 'n: !int "{{ 2 + 2 }}"' | uw config realize --input-format yaml --output-format yaml
n: 4
```
because it only takes effect later, after Jinja2 has been used to render the expression. So my suggestion here to say "use cases" instead of "formats" is because the format is `int` either way, but the use case is different.